mpmath/libmp/libmpf.py

```diff
@@ -1098,7 +1098,7 @@ def to_digits_exp(s, dps, base=10):
     exponent += len(digits) - fixdps - 1
     return sign, digits, exponent
 
-def round_digits(sign, digits, dps, base, rounding=round_nearest):
+def round_digits(sign, digits, dps, base, rounding=round_nearest, fixed=False):
```

Comment:
It seems to be the key.  Now I think that the fix should go to "Rounding up kills some instances" block and could be one-liner.